probe-rs/src/vendor/vorago/sequences/va416xx.rs

```diff
@@ -53,11 +53,23 @@ impl ArmDebugSequence for Va416xx {
 
         // Re-initializing the core(s) is on us.
         let ap = interface.ap();
+
```

Comment:
Can you please fix the `Ingore` in the comment a few lines up? :D

Reply:
Done